hash/to_field.go

```diff
@@ -0,0 +1,5 @@
+package hash
+
+type ToField interface {
+	WriteString(rawBytes []byte)
```

Comment:
I think this file is not needed; just do a private interface{} in `transcript.go`